source/plugin/data/serialization.rst

```diff
@@ -123,3 +125,56 @@ from persistent files using the :doc:`Configurate Library <../configuration/inde
         final DataView container = translator.translate(node);
         return deserializeHealth(container);
     }
+
+
+DataFormat
+==========
+An alternative to DataTranslators is to use :javadoc:DataFormat, which allows you to store a ``DataContainer`` as a hocon, json or NBT file. You can also retrieve data using ``DataFormat``.
+
+This is very useful if you're for example using a database to store information, as you can then serialize any ``DataContainer`` to json format.
+
+For this example we will use :javadoc:`DataFormats#JSON` to translate a player to json and back! 
+
+**Imports for code examples**
+    import org.spongepowered.api.data.DataContainer;
+    import org.spongepowered.api.data.persistence.DataFormat;
+    import org.spongepowered.api.data.persistence.DataFormats;
+    import org.spongepowered.api.entity.living.player.Player;
+
+    import java.io.ByteArrayOutputStream;
+    import java.io.IOException;
+    import java.io.OutputStream;
+    import java.io.InputStream;
+    import java.util.Optional;
+
+**Code Example: Serializing a Player to json format**
+    
+.. code-block:: java
+
+    public OutputStream toJsonFormat(Player player){
```

Comment:
i will use itemstack but i havnt written it to a file because that isnt really what i would expect people to do with it there more likly to use DataTranslators for that i was more thinking this would be used for databases etc also im not 100% sure how to go from datacontainer to the orignal object havnt been able to work that out yet